src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,94 @@
-import React from 'react';
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react';
+import { getMovie } from '../../api/api';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+type Props = {
+  addMovie: (newMovie: Movie) => void;
+};
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [title, setTitle] = useState('');
+  const [movie, setMovie] = useState(null);
+  const [error, setError] = useState(false);
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value.trimLeft());
+  };
+
+  const findMovieByTitle = () => {
+    getMovie(title).then(data => {
```

Comment:
unless a library or legacy codebase forces you to use then/catch, the better choice for readability and maintainability is async/await